streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -1243,6 +1245,50 @@ public void shouldRetryInitializationWhenLockExceptionInStateUpdater() {
         verify(stateUpdater).add(task01);
     }
 
+    @Test
+    public void shouldRetryInitializationWhenCanNotInitializeTask() {
+        final StreamTask task00 = statefulTask(taskId00, taskId00ChangelogPartitions)
+                .withInputPartitions(taskId00Partitions)
+                .inState(State.RESTORING).build();
```

Comment:
We use 4 spaces and not 8 for indentation.
```suggestion
            .withInputPartitions(taskId00Partitions)
            .inState(State.RESTORING).build();
```